licenses/BSD-3-Clause

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2013, Mitchell Bösecke
```

Comment:
How is this different from BSD?

Reply:
> How is this different from BSD?

java-ebuilder finds what's given in  https://github.com/PebbleTemplates/pebble/blob/3.1.6/pom.xml#L28. Missing `BSD-3-Clause` urges packager to search and adjust manually in the ebuild.

Reply:
Looks like maybe java-ebuilder needs to gain support for `metadata/license-mapping.conf`?

Reply:
> Looks like maybe java-ebuilder needs to gain support for `metadata/license-mapping.conf`?

https://bugs.gentoo.org/907208